sklearn/tests/test_pipeline.py

```diff
@@ -603,6 +603,14 @@ def test_make_union_kwargs():
         make_union(pca, mock, transformer_weights={"pca": 10, "Transf": 1})
 
 
+def test_make_union_passes_verbose_feature_names_out():
+    pca = PCA()
+    mock = Transf()
+
+    union = make_union(pca, mock, verbose_feature_names_out=False)
+    assert not union.verbose_feature_names_out
```

Comment:
Would it be possible to also have a case where `verbose_feature_names_out=True` and we can verify that the feature names are as expected?